Formula/mpd.rb

```diff
@@ -63,6 +65,7 @@ class Mpd < Formula
   depends_on "libvorbis" => :optional
   depends_on "libnfs" => :optional
   depends_on "mad" => :optional
+  depends_on "libmodplug" => :optional  # MODPlug decoder
```

Comment:
fwiw i think it's at least consistent with the lines further above, though some of these are more obvious than others